pkg/phlaredb/symdb/stacktrace_tree_test.go

```diff
@@ -220,9 +222,48 @@ func Benchmark_stacktrace_tree_insert(b *testing.B) {
 	b.ReportAllocs()
 
 	for i := 0; i < b.N; i++ {
-		x := newStacktraceTree(0)
+		x := newStacktraceTree(defaultStacktraceTreeSize)
```

Comment:
Just a note: `defaultStacktraceTreeSize` is 0.

Reply:
Yes. I just change it so if we ever adjust it, this original benchmark will be based on that value and not a hard coded 0. Would you like me to revert this change?

Reply:
No I don't think so. I think Anton just wanted to point out that this line is a no-op